tests/converter/test_code_generation.py

```diff
@@ -355,40 +364,21 @@ def codegen_cpu(self,
 
         self.assertTrue(os.path.exists(project_dir))
 
-        cmake_use_aarch64 = '-DTOOLCHAIN_NAME=linux_aarch64'
-        cmake_use_arm = '-DTOOLCHAIN_NAME=linux_arm'
-        cmake_use_neon = '-DUSE_NEON=1'
-        cmake_use_fpga = '-DRUN_ON_FPGA=1'
-        cmake_use_avx = '-DUSE_AVX=1'
-
-        cmake_defs = []
-        if cpu_name == 'aarch64':
-            cmake_defs += [cmake_use_aarch64, cmake_use_neon]
-        elif cpu_name == 'arm':
-            cmake_defs += [cmake_use_arm, cmake_use_neon]
-        elif cpu_name == 'arm_fpga':
-            cmake_defs += [cmake_use_arm, cmake_use_neon, cmake_use_fpga]
-        elif cpu_name == 'x86_64':
-            if use_avx:
-                cmake_defs += [cmake_use_avx]
-
-        run_and_check(['cmake'] + cmake_defs + ['.'],
+        run_and_check(['make', 'clean'],
                       project_dir,
-                      join(output_path, "cmake.out"),
-                      join(output_path, "cmake.err"),
-                      self,
-                      check_stdout_include=['Generating done'],
-                      check_stdout_block=['CMake Error']
+                      join(output_path, "make_clean.out"),
+                      join(output_path, "make_clean.err"),
+                      self
                       )
 
-        run_and_check(['make', 'VERBOSE=1', 'lib', '-j8'],
+        run_and_check(['make', 'lib_' + lib_base_name, '-j8'],
                       project_dir,
                       join(output_path, "make.out"),
                       join(output_path, "make.err"),
                       self,
-                      check_stdout_include=['Building'],
                       check_stderr_block=['error: ']
                       )
+
```

Comment:
@tk26eng 
Sorry for little thing.
We do not change code style fix with other implementation.